src/main/java/edu/carrollcc/cis232/Question1.java

```diff
@@ -6,5 +6,15 @@ public static void main(String[] args) {
 
 	}
 
-	//Write static maxElement method here
+	public static int maxElement(int[] array) {
+		if (array == null || array.length == 0) {
+			throw new IllegalArgumentException("Empty Array");
+		}
+		return maxElement(array, 0);
+	}
+
+	private static int maxElement(int[] array, int index) {
+		return index < array.length ? Math.max(array[index], maxElement(array, index + 1)) : Integer.MIN_VALUE;
```

Comment:
this is good, a little too good. Make sure you know exactly what is happening here